### PR TITLE
Update Autocompletion to point to the full release page for tools directory.

### PR DESCRIPTION
**Pull request by blhagadorn**
Addresses https://github.com/istio/istio/issues/21465
[ ] Configuration Infrastructure
[ x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by blhagadorn** [comment]
@esnible Could I get an ok to test?

File: content/en/docs/ops/diagnostic-tools/istioctl/index.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ To enable `istioctl` completion on your system, follow the steps for your prefer
 
 Installing the bash auto-completion file
 
-If you are using bash, the `istioctl` auto-completion file is located in the `tools` directory. To use it, copy the `istioctl.bash` file to your home directory, then add the following line to source the `istioctl` tab completion file from your `.bashrc` file:
+If you are using bash, the `istioctl` auto-completion file is located in the `tools` directory. To download the full release (which contains the `tools` directory), select the appropriate download from the [Istio Releases](https://github.com/istio/istio/releases) page. To use it, copy the `istioctl.bash` file to your home directory, then add the following line to source the `istioctl` tab completion file from your `.bashrc` file:
```
Comment (frankbu) [inline content/en/docs/ops/diagnostic-tools/istioctl/index.md]: This information is also needed in the ZSH tab, not just Bash. I suggest undoing this change and instead changing line 139 to the following:
```
To enable `istioctl` completion on your system, follow the steps for your preferred shell.

{{< warning >}}
You need to download the full Istio release containing the auto-completion files. If you haven't already done so [download the release](/docs/setup/getting-started/#download) now.
{{< /warning >}}
```

Comment (blhagadorn) [reply content/en/docs/ops/diagnostic-tools/istioctl/index.md]: Ahh good catch! Thanks @frankbu

**Pull request merged**